plugins/coral-plugin-love/server/typeDefs.graphql

```diff
@@ -0,0 +1,70 @@
+enum ACTION_TYPE {
+
+    # Represents a Love.
+    LOVE
```

Comment:
Ultimately my call is to not include the "CORAL-PLUGIN" everywhere. I tried doing that here: https://github.com/coralproject/talk-plugin-mentioned-in-story/blob/server-side-initial-dev/server/typeDefs.graphql#L45 and found that things are much clearer when the concept is modeled.